src/Rules/SyntaxErrorInDibiPreparedStatementMethodRule.php

```diff
@@ -112,13 +112,7 @@ private function checkErrors(CallLike $callLike, Scope $scope, MethodReflection
             $parameterType = $scope->getType($parameterExpr);
 
             if ($parameterType instanceof StringType) {
-                $resolvedString = $queryReflection->resolveQueryString($parameterExpr, $scope);
```

Comment:
looks like a unrelated change?

Reply:
Well, the original method `resolveQueryString` is deprecated, the suggested replacement `resolveQueryStrings` works, but the pass-through of the deprecated method is to a different method that is not doing what is expected and therefore is broken in this use case.

Basically, fixed something that was broken and would not allow any testing to pass in the first place, but was too lazy to do it in a separate stacked PR:

Reply:
I will move it into a separate PR, thank you